dao/src/test/resources/application.conf

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018-2019 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#system-wide time zone
+timezone="UTC"
```

Comment:
I see... it happens in `MenasPlainCredentialsSuite` which uses a `SparkSession`. Ideally it would not require any SparkSession, but that's some technical debt we can live with for now, I guess.